analyzers/tests/SonarAnalyzer.UnitTest/Helpers/ProjectConfigReaderTests.cs

```diff
@@ -103,49 +95,38 @@ public void WhenHasUnexpectedProjectType_FallsBackToProduct()
         [DataRow("MissingTargetFramework")]
         public void WhenHasMissingValues_FilesToAnalyzePath_ReturnsCorrectValue(string folder)
         {
-            var options = TestHelper.CreateOptions($"ResourceTests\\SonarProjectConfig\\{folder}\\SonarProjectConfig.xml");
-
-            var sut = new ProjectConfigReader(options);
+            var sut = CreateProjectConfigReader($"ResourceTests\\SonarProjectConfig\\{folder}\\SonarProjectConfig.xml");
 
             sut.FilesToAnalyzePath.Should().Be(@"c:\foo\bar\.sonarqube\conf\0\FilesToAnalyze.txt");
         }
 
-        [DataTestMethod]
-        [DataRow(null)]
-        [DataRow("/foo/bar/do-not-exit")]
-        [DataRow("/foo/bar/x.xml")]
-        public void WhenNoFileFound_ReturnsNull(string folder)
-        {
-            var options = TestHelper.CreateOptions(folder);
-
-            var sut = new ProjectConfigReader(options);
-
-            sut.AnalysisConfigPath.Should().BeNull();
-            sut.ProjectPath.Should().BeNull();
-            sut.FilesToAnalyzePath.Should().BeNull();
-            sut.OutPath.Should().BeNull();
-            sut.ProjectType.Should().Be(ProjectType.Product);
-            sut.TargetFramework.Should().BeNull();
-        }
-
         [DataTestMethod]
         [DataRow("Invalid_DifferentClassName")]
         [DataRow("Invalid_DifferentNamespace")]
         [DataRow("Invalid_Xml")]
         public void WhenInvalid_FilesToReturnPath_ThrowsException(string folder)
         {
-            var options = TestHelper.CreateOptions($"ResourceTests\\SonarProjectConfig\\{folder}\\SonarProjectConfig.xml");
+            Action a = () => CreateProjectConfigReader($"ResourceTests\\SonarProjectConfig\\{folder}\\SonarProjectConfig.xml");
```

Comment:
With Assert.ThrowsException it can be simplified.
```suggestion
Assert.ThrowsException<InvalidOperationException>(() => CreateProjectConfigReader($"ResourceTests\\SonarProjectConfig\\{folder}\\SonarProjectConfig.xml"))
                                                             .Message.Should().Be("File LogNameFor-SonarProjectConfig.xml could not be parsed.");
```